apps/jumpserver/conf.py

```diff
@@ -488,7 +488,6 @@ class Config(dict):
         'LOGIN_REDIRECT_TO_BACKEND': '',  # 'OPENID / CAS / SAML2
         'LOGIN_REDIRECT_MSG_ENABLED': True,
 
-
         # 人脸识别
         'FACE_RECOGNITION_ENABLED': False,
         'FACE_RECOGNITION_DISTANCE_THRESHOLD': 0.35,
```

Comment:
I'm sorry, but I can't see any code provided for checking differences. Please provide more context or share specific diffs between the two versions of the config dictionary to be able to help you with your request.